helix-view/src/editor.rs

```diff
@@ -746,6 +756,16 @@ pub enum CloseError {
     SaveError(anyhow::Error),
 }
 
+/// The current search position
+pub struct SearchPosition {
+    /// The position of the current match
+    pub current_position: usize,
+    /// The total number of matches
+    pub total_positions: usize,
+    /// True if we had wrapped
+    pub wrapped: bool,
+}
```

Comment:
Search position should be stored on the Document and not the Editor: the statusline element becomes inaccurate when you switch between splits

Reply:
done